sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_ga_to_subscriptions_v1/query.sql

```diff
@@ -0,0 +1,163 @@
+WITH website_base AS (
+  SELECT
+    `date`,
+    mozfun.norm.vpn_attribution(
+      NULL, -- provider
+      NULL, -- referrer
+      campaign,
+      content,
+      medium,
+      source
+    ).*,
+    sessions,
+    subscribe_intent_goal,
+    country,
+  FROM
+    site_metrics_summary_v1
+  WHERE
+    -- Populate from mozilla.org starting 2021-03-11, and from vpn.mozilla.org before that
+    IF(`date` < DATE "2021-03-11", site = "vpn.mozilla.org", site = "mozilla.org")
+),
+website AS (
+  SELECT
+    `date`,
+    IF(
+      COUNT(DISTINCT COALESCE(normalized_acquisition_channel, "NULL")) = 1,
+      ANY_VALUE(normalized_acquisition_channel),
+      ERROR("normalized_acquisition_channel must have one value per group")
+    ) AS normalized_acquisition_channel,
+    normalized_medium,
+    normalized_source,
+    normalized_campaign,
+    normalized_content,
+    IF(
+      COUNT(DISTINCT COALESCE(website_channel_group, "NULL")) = 1,
+      ANY_VALUE(website_channel_group),
+      ERROR("website_channel_group must have one value per group")
+    ) AS website_channel_group,
+    SUM(sessions) AS sessions,
+    SUM(
+      IF(
+        country IN (
```

Comment:
Maybe stick this in a temp udf at the top so it only needs to be changed once when DE and FR are added.